pandas/core/arrays/base.py

```diff
@@ -403,13 +402,14 @@ def argsort(self, ascending=True, kind='quicksort', *args, **kwargs):
         """
         # Implementor note: You have two places to override the behavior of
         # argsort.
-        # 1. _values_for_argsort : construct the values passed to np.argsort
+        # 1. _values_for_argsort : construct the values passed to np.argsort.
         # 2. argsort : total control over sorting.
         ascending = nv.validate_argsort_with_ascending(ascending, args, kwargs)
+
         values = self._values_for_argsort()
-        result = np.argsort(values, kind=kind, **kwargs)
-        if not ascending:
-            result = result[::-1]
+        na_position = 'last' if ascending else 'first'
```

Comment:
so we actually expose ``na_position`` as a parameter (defaults to 'last') in ``.sort_values()`` is there a reason we should not do the same here? @TomAugspurger @jorisvandenbossche 

also this should be independent of ascending (e.g. should default to 'last')

Reply:
I think @jorisvandenbossche's preference was to not add an argument, as that complicates the implementation for every EA.

Reply:
ok that's reasonable